lxd/container.go

```diff
@@ -41,10 +41,9 @@ import (
 )
 
 func init() {
-	// Expose instanceLoadNodeAll to the device package converting the response to a slice of Instances.
```

Comment:
@stgraber did you intend to get ride of this comment line?

Reply:
Nope, will fix